res/css/photon/button.css

```diff
@@ -4,9 +4,12 @@
 
 /* See https://design.firefox.com/photon/components/buttons.html for the spec */
 .photon-button {
-  /* These two flex options aren't necessary when a real <button> is used, but they are when a <a> element is used as abutton */
-  display: flex;
+  /* These flex and sizing properties aren't necessary when a real <button> is used,
+   * but they are when a <a> element is used as a button. */
+  display: inline-flex;
+  box-sizing: border-box;
   align-items: center;
+  justify-content: center;
```

Comment:
The button have `text-align: center` but that doesn't work anymore once we move it to flex. Witth `justify-content: center` this should do the same thing in flex context.